src_tests/test_simulation_strategies.py

```diff
@@ -302,18 +316,21 @@ def __test_haar_random_interferometers_approximation_distance_from_ideal(self,
         )
 
         probabilities_list = []
+        current_probabilities = []
```

Comment:
Why predefining this array if it's not used after the `for` loop? Is there something I'm missing here?

Reply:
But it is - line 340.

Reply:
Oh, it was a part of the folded code, so I missed it.